File: array.c
#include <stdio.h>

int main() {
    int arr[5] = {10, 20, 30, 40, 50};

    printf("Base address of arr: %p\n", arr); // Base address of array

    printf("1D Array Traversal:\n");
    for (int i = 0; i < 5; i++) {
        printf("arr[%d] = %d at address %p\n", i, arr[i], &arr[i]);
    }

    return 0;
}

File: Array.java
public class Array {
    public static void main(String[] args) {
        int[] arr = {10, 20, 30, 40, 50};

        System.out.println("Base address (hashcode) of arr: " + arr);

        System.out.println("1D Array Traversal:");
        for (int i = 0; i < arr.length; i++) {
            System.out.println("arr[" + i + "] = " + arr[i]);
        }
    }
}
